### PR TITLE
fix(grid): Change gutter size to be usable as a multiple of 4

### PR DESCRIPTION
**Pull request by esraltintas**
### Description

- Whenever we defined the gutter spacings as 12 for the all devices in the grid, it didn't seem to add the required classes to the row

Screenshot 2022-08-26 at 11 16 10

### Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation change

### How do I test this

- Add steps to test
- in bullet point format
- preferably you can add link to the storybook build in the PR

### Did you remember to take care of the following?

- [x] `npm i` – for new NPM dependencies.
- [x] `npm run lint` - to check for linting issues
- [x] `npm run test` - to run unit tests
- [x] `npm run test:screenshots` - to run snapshot tests

### New Feature / Bug Fix

- [x] Run unit tests to ensure all existing tests are still passing.
- [x] Add new passing unit tests to cover the code introduced by your pr.

Thanks for contributing!

### CODE REVIEW

File: packages/grid/src/utils/getGridClassNames.test.tsx [hunk 1]
```diff
@@ -33,7 +46,15 @@ describe('Grid utils', () => {
     ).toEqual([
       'order-1_prefix',
       'width-auto_prefix',
-      'gutter-mobile-1_prefix',
+      undefined,
```
Comment (derek-heycar) [inline packages/grid/src/utils/getGridClassNames.test.tsx:49]: As I understand we should avid `undefined `, why in this case is ok if result equal  to `undefined ` ?

**Review by derek-heycar — CHANGES_REQUESTED** [review]

File: packages/grid/src/utils/getGridClassNames.test.tsx (hunk 1 above)
Comment (esraltintas) [reply packages/grid/src/utils/getGridClassNames.test.tsx:49]: @derek-heycar In this test, we are testing dimensions, if anybody set any dimension less than "4" they can get "undefined", we have another task to improve `getGridClassNames` utils test, I'll open another pr after merging this pr

Comment (derek-heycar) [reply packages/grid/src/utils/getGridClassNames.test.tsx:49]: But what is the meaning of this test case? 
Does not matter what you add this test will passed ?

Comment (esraltintas) [reply packages/grid/src/utils/getGridClassNames.test.tsx:49]: @derek-heycar Please check this pull request, I resolved this in this pr https://github.com/hey-car/heycar-uikit/pull/72/files

**Pull request closed without merging**